### PR TITLE
Allow running multiple bootstrap commands in Krb

### PR DESCRIPTION
**Pull request by adamjshook**
Supersedes #56 

Co-authored-by: Łukasz Walkiewicz <lukasz.walkiewicz@starburstdata.com>

### CODE REVIEW

**Comment by adamjshook** [comment]
@kokosing Do we want to keep the existing variable for backwards compatibility? Or assume updating to the new Kerberos container uses the new method?

File: prestodev/kerberos/files/opt/entrypoint.sh [hunk 1]
```diff
@@ -4,8 +4,10 @@ set -exuo pipefail
 
 "$@" &
 
-if [[ -v PRESTODEV_POST_BOOTSTRAP_COMMAND ]]; then
-    $PRESTODEV_POST_BOOTSTRAP_COMMAND
+if test -d /docker/presto-init.d; then
```
Comment (kokosing) [inline prestodev/kerberos/files/opt/entrypoint.sh:7]:
```suggestion
if test -d /docker/kerberos-init.d; then
```
?

**Review by kokosing — APPROVED** [review]

File: prestodev/kerberos/files/opt/entrypoint.sh [hunk 2]
```diff
@@ -4,8 +4,10 @@ set -exuo pipefail
 
 "$@" &
 
-if [[ -v PRESTODEV_POST_BOOTSTRAP_COMMAND ]]; then
```
Comment (kokosing) [inline prestodev/kerberos/files/opt/entrypoint.sh:7]: > Do we want to keep the existing variable for backwards compatibility? Or assume updating to the new Kerberos container uses the new method?

Yes. It is much easier when you just need a single principal.

Comment (adamjshook) [reply prestodev/kerberos/files/opt/entrypoint.sh]: Done!

**Review by kokosing — APPROVED** [review]

**Comment by kokosing** [comment]
Merged, thanks!

**Pull request merged**